redbot/core/core_commands.py

```diff
@@ -1634,18 +1642,21 @@ async def leave(self, ctx: commands.Context, *servers: GuildConverter):
             if pred.result is True:
                 if leaving_local_guild is True:
                     await ctx.send(_("Alright. Bye :wave:"))
+                if number > 1:
+                    await ctx.send(_(f"Alright. Leaving {number} servers..."))
```

Comment:
F-strings cannot be used for translatable strings.
```suggestion
                    await ctx.send(
                        _("Alright. Leaving {number} servers...").format(number=number)
                    )
```